blueocean-core-js/src/js/services/PipelineService.js

```diff
@@ -25,27 +25,16 @@ export class PipelineService extends BunkerService {
     }
 
     /**
-     * Gets pager for /blue/pipelines
-     *
-     * @returns {Pager}
-     */
-    allPipelinesPager() {
```

Comment:
Yeah, this should not be in core-js - it just seems to be providing data for a specific page...